### DISCUSSION TITLE
How can I remove '1' (or 0) text in TextColumn in Table builder?

### DISCUSSION THREAD

**Discussion by itbaracoal** [filamentphp/filament#14994] [Help] [answered]
### Package

Table builder

### Package Version

v3.2

### How can we help you?

![image](https://github.com/user-attachments/assets/ec23cb07-5347-46b4-a5fd-9ea33a4b9b4f)

my code:

```
Tables\Columns\TextColumn::make('check_list_dokumen')
                    ->label('Checklist')
                    ->icon(function ($state) {
                        return $state
                            ? 'heroicon-o-check-circle'
                            : 'heroicon-o-x-circle';
                    })
                    ->iconColor(function ($state) {
                        return $state ? 'success' : 'danger';
                    }),
```

I can't remove the extra text after the icon, I want only the icon appear. 
how to do that? thank you.

**Comment by leandrocfe** [comment, marked as answer]
Just use [IconColumn](https://filamentphp.com/docs/3.x/tables/columns/icon)

**Reply by itbaracoal** [reply]
Ah, just as I wanted. Thank you🙌

**Reply by itbaracoal** [reply]
> Just use [IconColumn](https://filamentphp.com/docs/3.x/tables/columns/icon)

Hi Leandro,
I hope you are doing well.
I have another question regarding the icon, which currently uses py-4 as the default padding. How can I remove the padding or reduce the space? Or, if possible, how can I apply a global style?

```
Tables\Columns\IconColumn::make('check_list_dokumen')
    ->label('Checklist')
    ->boolean()
    ->extraAttributes(['class' => 'py-1']),
```

I have tried adding extraAttributes, but it doesn't seem to work.
thank you.

**Reply by leandrocfe** [reply]
Create a [custom theme](https://filamentphp.com/docs/3.x/panels/themes/#creating-a-custom-theme)
